site/docs/v1/tech/messengers/generic-messenger.adoc

```diff
@@ -11,17 +11,17 @@ Available since 1.26.0
 
 == Set Up a Generic Messenger
 
-Currently, messengers can only be configured to send messages via a SMS protocol. If you need to connect to an SMS provider other than Twilio, you can use the Generic messenger configuration.
+Currently, built-in messengers can only be configured to send messages via a Twilio. If you need to connect to a provider other than Twilio, whether you want to send SMS messages or any other, you can use the Generic messenger configuration.
 
-The generic messenger allows for a simple JSON REST API integration which allows you to receive a message from FusionAuth and then make an API call to any third party SMS provider you wish.
+The generic messenger allows for a simple JSON REST API integration which allows you to receive a message from FusionAuth to an arbitrary URL. The Generic Message Receiver at this URL may then make an API call to any third party messaging provider.
```

Comment:
I am not sure if this is needed, but a thought:

> In other words, a Generic Messenger is a thin coordination layer between FusionAuth and other messaging services (with the proper authority and functionality to act on a message).

Reply:
Thanks, I tweaked it a bit, but I like the 'thin coordination layer' verbiage.